staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -217,7 +217,7 @@ type VolumeSource struct {
 	// The types of objects that may be mounted by this volume are defined by the container runtime implementation on a host machine and at minimum must include all valid types supported by the container image field.
 	// The OCI object gets mounted in a single directory (spec.containers[*].volumeMounts.mountPath) by merging the manifest layers in the same way as for container images.
 	// The volume will be mounted read-only (ro) and non-executable files (noexec).
-	// Sub path mounts for containers are not supported (spec.containers[*].volumeMounts.subpath).
+	// Sub path mounts for containers are not supported (spec.containers[*].volumeMounts.subpath) before 1.33.
```

Comment:
if KAS has drifted from kubelet and the former supports but the latter doesn't yet is that an issue? the pod will fail at creation time I presume but is that UX alright? Anything we can do?

Reply:
If the API server allows the sub path but the kubelet not, then the kubelet will ignore it. 
If the kubelet supports the sub path but the runtime not, then the runtime will ignore it.